xtp/src/pyxtp/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ if(NOT PYTHON_INSTDIR)
   execute_process(
     COMMAND
       ${Python_EXECUTABLE} -c
-      "import distutils.sysconfig as cg; print(cg.get_python_lib(prefix='${CMAKE_INSTALL_PREFIX}', plat_specific=True, standard_lib=False))"
+      "import distutils.sysconfig as cg; print(cg.get_python_lib(plat_specific=True, standard_lib=False))"
```

Comment:
When in a conda env prefix was still pointing at the base Python install and the module was the. Not installed properly. There might be a better solution to insure the module is always installed where the user expect it to be … ?

Reply:
I have the feeling the issue might have been setting an incorrect `CMAKE_INSTALL_PREFIX` since here that is used directly, no other source of variation.  Maybe I'll experiment a bit later how this changes under conda.

Reply:
I think the [`cmake` default for the prefix](https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX.html) is actually reasonable. It fails because it's in a `conda` environment.  There might be a way to [set it differently](https://cmake.org/cmake/help/latest/variable/CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT.html) in a conda environment.  I could have a look.